### PR TITLE
Gwms frontend

### PR DESCRIPTION
**Pull request by efajardo**
https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/InstallGlideinWMSFrontend

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Add migration header to TWiki document

### CODE REVIEW

**Comment by efajardo** [comment]
@brianhlin can you review and merge please?

File: docs/other/install-gwms-frontend.md [hunk 1]
```diff
@@ -0,0 +1,983 @@
...
+---------------------------------------------------------------------------
+
+You must report to Gratia if you are running on OSG more than a few test jobs.
+
+[ProbeConfigGlideinWMS](https://twiki.opensciencegrid.org/bin/view/Accounting/ProbeConfigGlideinWMS) explains how to instal and configure the HTCondor Gratia probe. If you are on a Campus Grid without x509 certificates pay attention to the [Users without Certificates part](https://twiki.opensciencegrid.org/bin/view/Accounting/ProbeConfigGlideinWMS#Users_without_Certificates) in the Unusual Use Cases section.
+
+Optional Configuration
+----------------------
+
+The following configuration steps are optional and will likely not be required for setting up a small site. If you do not need any of the following special configurations, skip to [the section on service activation/deactivation](#ServiceActivation).
```
Comment (matyasselmeci) [inline docs/other/install-gwms-frontend.md]: Broken links here and in the list

File: docs/other/install-gwms-frontend.md [hunk 2]
```diff
@@ -0,0 +1,983 @@
...
+- [Creating a group to test configuration changes](#IsItb)
+
+### Allow users to specify where their jobs run
+
+In order to allow users to specify the sites at which their jobs want to run (or to test a specific site), a frontend can be configured to match on `DESIRED_Sites` or ignore it if not specified. Modify `/etc/gwms-frontend/frontend.xml` using the following instructions:
+
+1. In the frontend's global `<match>` stanza, set the `match_expr`:
+
+        :::file
+        '((job.get("DESIRED\_Sites","nosite")=="nosite") or (glidein\["attrs"\]\["GLIDEIN\_Site"\] in job.get("DESIRED\_Sites","nosite").split(",")))'
```
Comment (matyasselmeci) [inline docs/other/install-gwms-frontend.md]: Fix `\` before `_`, `[`, and `]`.

File: docs/other/install-gwms-frontend.md [hunk 3]
```diff
@@ -0,0 +1,983 @@
...
+
+2. In the same `<match>` stanza, set the `start_expr`:
+
+        :::file
+        '(DESIRED_Sites`?=undefined || stringListMember(GLIDEIN\_Site,DESIRED\_Sites,","))
```
Comment (matyasselmeci) [inline docs/other/install-gwms-frontend.md]: Fix `\_`. Also, `` `?=  `` should be `=?=`.

File: docs/other/install-gwms-frontend.md [hunk 4]
```diff
@@ -0,0 +1,983 @@
...
+Certificates brief
+------------------
+
+Here a short list of files to check when you change the certificates. Note that if you renew a proxy or certificate and the DN remains the same no configuration file needs to change, just put the renewed certificate/proxy in place.
+
+|              File Description             |                               File Location                               |
+|:-----------------------------------------:|:-------------------------------------------------------------------------:|
+|             Configuration file            |                      /etc/gwms-frontend/frontend.xml                      |
+|         HTCondor certificates map         |                   /etc/condor/creds/condor\_mapfile (1)                   |
+|        Host certificate and key (2)       | /etc/grid-security/hostcert.pem&lt;br/&gt; /etc/grid-security/hostkey.pem |
```
Comment (matyasselmeci) [inline docs/other/install-gwms-frontend.md]: Get rid of the `&lt;br/&gt;` -- it doesn't work. split this into two rows instead.

**Comment by efajardo** [comment]
Thanks for reviewing @matyasselmeci I made the changes oyu requested, can you check now please?

**Pull request merged**